profiles/default.yml

```diff
@@ -131,6 +133,9 @@ scripts:
     dotty:
         - measure $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
 
+    dotty-sbt:
+        - measure -Yforce-sbt-phases $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
+
```

Comment:
@allanrenucci Just to confirm, testing dotty is extremely expensive (1 hour), it seems we can also test other projects with the same option.